terraform, gcp: add opt-out var to collaboratively and opportunistically apply node-purpose infra labels

consideRatio opened the pull request:
This is for #3405, a followup to #3397 that is work I think all @2i2c-org/engineering needs to opportunistically apply over time when given the chance. @2i2c-org/engineering the ask is that when you apply terraform config, consider if there are a fixme that you'd like to opportunistically resolve while you are at it.

This PR adds variables and fixme notes in individual GCP clusters' .tfvar files, allowing us to see where we haven't yet managed to apply what #3397 introduced.

The purpose of this is to:
- ensure our terraform state is reflect in our config
- set us up to collectively and opportunistically work towards the desired state over time
- unblock other changes that may not be disruptive in the terraform config

yuvipanda reviewed (approved):
This is pretty clever, thanks for doing this @consideRatio.

I've just applied tf to pangeo-hubs (and ticked it off in the PR I was using to track) so can you remove that from here before merging?

consideRatio commented:
> I've just applied tf to pangeo-hubs (and ticked it off in the PR I was using to track) so can you remove that from here before merging?

Ah nice @yuvipanda, I rebased and removed that commit!

consideRatio commented:
Thank you for reviewing and working on this @yuvipanda!!

consideRatio commented:
Argh, removed `pilot-hubs` instead of `pangeo-hubs`... =/ I'll open a new PR to fix it.
